tests/acceptance/expected-failures-API-on-OWNCLOUD-storage.md

```diff
@@ -2041,7 +2041,7 @@ _ocs: api compatibility, return correct status code_
 -   [apiProvisioningGroups-v2/getSubAdminGroups.feature:11](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiProvisioningGroups-v2/getSubAdminGroups.feature#L11)
 -   [apiProvisioningGroups-v2/getSubAdminGroups.feature:25](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiProvisioningGroups-v2/getSubAdminGroups.feature#L25)
 -   [apiProvisioningGroups-v2/getSubAdminGroups.feature:34](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiProvisioningGroups-v2/getSubAdminGroups.feature#L34)
--   [apiProvisioningGroups-v2/getSubAdminGroups.feature:51](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiProvisioningGroups-v2/getSubAdminGroups.feature#L51)
+-   [apiProvisioningGroups-v2/](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiProvisioningGroups-v2/getSubAdminGroups.feature#L51)
```

Comment:
This looks like an accident

Reply:
I don't know how it got erased. I'll fix it.